src/main_dpdk.cpp

```diff
@@ -5951,6 +5978,24 @@ COLD_FUNC int update_global_info_from_platform_file(){
         }
     }
 
+    if (cg->m_latency_measurement.length()) {
+        if ((strcasecmp(cg->m_latency_measurement.substr(0, 5).c_str(), "NANOS") == 0) ||
```

Comment:
Inconsistency between those two methods.
In one you use .substr() in other not, why?

Reply:
I will check the whole string